crates/generate_artifacts/src/iso_overload_file.rs

```diff
@@ -46,13 +46,24 @@ fn build_iso_overload_for_client_defined_type<TNetworkProtocol: NetworkProtocol>
 ) -> (String, String) {
     let (client_type, variant) = client_type_and_variant;
     let mut s: String = "".to_string();
-    let import = format!(
+    let mut import = format!(
         "import {{ type {}__param }} from './{}/{}/param_type{}';\n",
         client_type.type_and_field().underscore_separated(),
         client_type.type_and_field().type_name,
         client_type.type_and_field().field_name,
         file_extensions.ts()
     );
+
+    if let SelectionType::Object(client_pointer) = client_type {
+        import = format!(
+            "{}import {{ type {}__link__output_type }} from './{}/link/output_type{}';\n",
```

Comment:
will we import the same link multiple times if there are multiple pointers to the same type?